docs/common/craft-parts/reference/plugins/scons_plugin.rst

```diff
@@ -0,0 +1,61 @@
+.. _craft_parts_scons_plugin:
+
+Scons plugin
+============
+
+The Scons plugin builds projects using Scons_.
+
+After a successful build, this plugin will installs the generated
+binaries in ``$CRAFT_PART_INSTALL``.
+
+Keywords
+--------
+
+In addition to the common :ref:`plugin <part-properties-plugin>` and
+:ref:`sources <part-properties-sources>` keywords, this plugin provides the following
+plugin-specific keywords:
+
+scons-parameters
+~~~~~~~~~~~~~~~~
+**Type:** list of strings
+**Default:** []
+
+Parameters to pass to scons for building and installation.
+
+Environment variables
+---------------------
+
+This plugin sets ``DESTDIR`` to ``$CRAFT_PART_INSTALL``.
+
+Dependencies
+------------
+
+The Scons plugin needs the ``scons`` executable work.
```

Comment:
should also mention that the plugin itself does NOT provision scons

Reply:
yeah, I recalled this when I was picking up my kids. I was wondering why I had not submitted it by then. thanks!